app/controllers/items/new.js

```diff
@@ -76,6 +76,14 @@ export default GoodcityController.extend(
     }),
     offersLists: [],
 
+    saleableId: Ember.computed("saleableOptions", function() {
+      return this.get("saleableOptions").get("firstObject");
+    }),
+
+    restrictionId: Ember.computed("restrictionOptions", function() {
+      return this.get("restrictionOptions").get("firstObject");
+    }),
```

Comment:
Since you're modifying those, it's better not to use a computed property, you can simply initialise them